src/py/_pyodide/_core_docs.py

```diff
@@ -1272,7 +1305,7 @@ def default_converter(_value, _ignored1, _ignored2):
 
         class Pair:
             def __init__(self, first, second):
-                self.first = first self.second = second
+                self.first, self.second = first, second
```

Comment:
Thanks for fixing this! I think it should read:
```suggestion
                self.first = first
                self.second = second
```

Reply:
Could you split the assignment back onto two lines?